File: Day 1 Data Types/Day 1 Data Types.c
#include <iostream>
#include <iomanip>
#include <limits>

using namespace std;

int main() {
    int i = 4;
    double d = 4.0;
    string s = "HackerRank ";

    
    // Declare second integer, double, and String variables.
    int second_integer;
    double second_double;
    string second_string;
    // Read and save an integer, double, and String to your variables.
    string temp;
    getline(cin, temp);
    second_integer = stoi(temp);
    getline(cin, temp);
    second_double = stod(temp);
    getline(cin, second_string);
    // Note: If you have trouble reading the entire string, please go back and review the Tutorial closely.
    
    // Print the sum of both integer variables on a new line.
    cout<<i+second_integer<<endl;
    // Print the sum of the double variables on a new line.
    cout<<fixed<<setprecision(1)<<d+second_double<<endl;
    // Concatenate and print the String variables on a new line
    // The 's' variable above should be printed first.
    cout<<s<<second_string<<endl;
    return 0;

File: Day 1 Data Types/Day 1 Data Types.java
import java.io.*;
import java.util.*;
import java.text.*;
import java.math.*;
import java.util.regex.*;

public class Solution {

    public static void main(String[] args) {
        int i = 4;
        double d = 4.0;
        String s = "HackerRank ";

        Scanner scan = new Scanner(System.in);

        /* Declare second integer, double, and String variables. */
        int second_integer;
        double second_double;
        String second_string;

        /* Read and save an integer, double, and String to your variables. */
        // Note: If you have trouble reading the entire String, please go back and
        // review the Tutorial closely.
        second_integer = scan.nextInt();
        second_double = scan.nextDouble();
        scan.nextLine();
        second_string = scan.nextLine();
        /* Print the sum of both integer variables on a new line. */
        System.out.println(i + second_integer);
        /* Print the sum of the double variables on a new line. */
        System.out.println(d + second_double);
        /*
         * Concatenate and print the String variables on a new line;
         * the 's' variable above should be printed first.
         */
        System.out.println(s.concat(second_string));
        scan.close();
    }
}